lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -36,7 +36,10 @@
       % if (cert_status['status'] == 'generating' or cert_status['status'] == 'downloadable'):
         <p class="message-copy">
           <%
-            certificate_available_date_string = course_overview.certificate_available_date.strftime('%Y-%m-%dT%H:%M:%S%z')
+            if course_overview.certificate_available_date:
+              certificate_available_date_string = course_overview.certificate_available_date.strftime('%Y-%m-%dT%H:%M:%S%z')
+            else:
+              certificate_available_date_string = course_overview.end.strftime('%Y-%m-%dT%H:%M:%S%z')
```

Comment:
We haven’t always enforced end dates on course runs. It’s not safe to assume it exists. If the certificate availability date is not set, we shouldn’t display anything.